roles/vdbench/templates/servers.yaml

```diff
@@ -25,7 +25,7 @@ spec:
               topologyKey: "kubernetes.io/hostname"
       containers:
       - name: vdbench-server
-        image: "quay.io/avili/vdbench:latest"
+        image: "quay.io/avili/vdbench:1.0.1"
```

Comment:
I think you mean for this to be quay.io/cloud-bulldozer/vdbench:master to be consistent with other benchmarks.    Your CI script can change the tag to snafu_ci in this file as part of the CI test, see other example *_wrapper/ci_test.sh files.  When developing, you can temporarily edit this to point to your own account's vdbench image.